ui/client/components/graph/EspNode/esp.ts

```diff
@@ -1,15 +1,16 @@
 /* eslint-disable i18next/no-literal-string */
-import * as joint from "jointjs"
-import {dia, util, shapes} from "jointjs"
+import {dia, shapes, util} from "jointjs"
 import expandIcon from "../../../assets/img/expand.svg"
 import "../graphTheme.styl"
+import {getRoundedRectPath} from "./getRoundedRectPath"
 
 const CONTENT_COLOR = "#1E1E1E"
 const PORT_COLOR = "#FFFFFF"
 const BORDER_COLOR = "#B5B5B5"
 
 const rectWidth = 300
 const rectHeight = 60
+const BORDER_RADIUS = 5
```

Comment:
Can we normailze all const to underscore coding standard?  Yes, I know it's not part of your change but still.. It's ugly :/